HEC.FDA.ViewModel/Hydraulics/GriddedData/GriddedImporterVM.cs

```diff
@@ -174,25 +194,25 @@ public override void Save()
             }
         }
 
+        /// <summary>
+        /// Checks whether the Name has changed in the UI and if so, renames the directory in the study.
+        /// </summary>
         private void RenameDirectoryInTheStudy()
         {
-            if (!Name.Equals(OriginalElement.Name))
-            {
-                string sourceFilePath = Connection.Instance.HydraulicsDirectory + "\\" + OriginalElement.Name;
-                string destinationFilePath = Connection.Instance.HydraulicsDirectory + "\\" + Name;
-                Directory.Move(sourceFilePath, destinationFilePath);
-            }
+            if (Name.Equals(OriginalElement.Name)) { return; }
+            string sourceFilePath = Connection.Instance.HydraulicsDirectory + "\\" + OriginalElement.Name;
```

Comment:
Stylistic change. for voids, I generally prefer to early exit with if statements, rather than put all the logic in the if statement. keeps ups one set of brackets deep.